doc/materials.md

```diff
@@ -1,3 +1,8 @@
 # Materials API
 
-consult `/src/materials/minetest.lua` at this time
\ No newline at end of file
+The materials can be accessed anywhere in your mod with `xcompat.materials.material_name`, e.g. `xcompat.materials.steel_ingot` for whichever string name would closest represent the steel_ingot material in the current game.
+
+
+The `xcompat.materials` variable is changed to contain the correct material names for whichever game is launched. See the contents of `/src/materials/` for supported games and materials. 
```

Comment:
please point people to the table at https://github.com/mt-mods/xcompat/tree/master?tab=readme-ov-file#directly-supported-games-and-mods internals may (not intending to) change as long as the api stays the same